django_celery_results/models.py

```diff
@@ -136,6 +144,13 @@ class ChordCounter(models.Model):
         )
     )
 
+    class Meta:
```

Comment:
curious why two class meta?

Reply:
oops, thats a typo!

Reply:
I thought I understood.. but I dont see the second "class meta" you're referring to

Reply:
i was my mistake :D